templates/solutions/infrastructure/private-cloud-pricing.html

```diff
@@ -0,0 +1,521 @@
+    citation_source_name_text="Tim Rosenfield",
+    citation_source_title_text="CEO and Co-Founder,",
+    citation_source_organisation_text="Firmus",
+    is_shallow=True
+    ) -%}
+
+    {%- if slot == 'signpost_image' -%}
+      {{ image(url="https://assets.ubuntu.com/v1/61ddd2f2-firmus.png",
+            alt="Firmus logo",
+            width="568",
+            height="69",
+            hi_def=True,
+            loading="lazy") | safe
+      }}
+    {%- endif -%}
+
+    {%- if slot == 'cta' -%}
+      <a href="https://ubuntu.com/engage/kubernetes-openstack-case-study-firmus"
+         aria-label="Read the Firmus openstack case study">Read the case study&nbsp;&rsaquo;</a>
+    {%- endif -%}
+
+  {% endcall -%}
+
+  {% call(slot) vf_quote_wrapper(
+    quote_size="small",
+    quote_text="Canonical's solution was a third of the price of the other proposals we'd received. It is also proving to be highly cost-effective, both from CAPEX and OPEX perspectives.",
+    citation_source_name_text="Georgi Georgiev",
+    citation_source_title_text="CIO,",
+    citation_source_organisation_text="SBI BITS",
+    ) -%}
+
+    {%- if slot == 'signpost_image' -%}
+      {{ image(url="https://assets.ubuntu.com/v1/ac021a67-sbi-bits-logo-2.png",
+            alt="SBI BITS logo",
+            width="852",
+            height="267",
+            hi_def=True,
+            loading="lazy") | safe
+      }}
+    {%- endif -%}
+
+    {%- if slot == 'cta' -%}
+      <a href="https://ubuntu.com/engage/sbi-casestudy"
+         aria-label="Read the SBI BITS case study">Read the case study&nbsp;&rsaquo;</a>
+    {%- endif -%}
+
+  {% endcall -%}
+
+  <section class="p-section--deep">
+    <hr class="p-rule is-fixed-width" />
+    <div class="row--50-50">
+      <div class="col">
+        <h2>Related blogs</h2>
+      </div>
+      <div class="col">
+        <h3 class="p-heading--5">
+          <a href="/blog/aws-pricing">Have you ever wondered how AWS pricing actually works?</a>
+        </h3>
+        <p>
+          AWS pricing differs depending on the service mode. Amazon offers various options: from free tier to dedicated hosts. Learn how AWS pricing works in details.
+        </p>
+        <hr class="p-rule--muted" />
+        <h3 class="p-heading--5">
+          <a href="/blog/azure-pricing">Azure pricing explained</a>
+        </h3>
+        <p>
+          Azure pricing differs depending on the service mode. Azure offers various options: from free tier to reservations. Learn how Azure pricing works in details.
+        </p>
+        <hr class="p-rule--muted" />
+        <h3 class="p-heading--5">
+          <a href="/blog/gcp-pricing">How GCP pricing works</a>
+        </h3>
+        <p>
+          GCP pricing differs depending on the service mode. Google offers various options: from free trial to sole-tenant nodes. Learn how GCP pricing works.
+        </p>
+      </div>
+    </div>
+  </section>
+
+  {# djlint: off #}
+  <script>
```

Comment:
Possibly silly question here, but the limit seems quite high when compared to the actual limit where savings are observed. The max savings amount of 38% is reached at 127 machines

Reply:
I have raised this with the owners of the data set and am awaiting their response.